analysis/grammar_search_analysis.py

```diff
@@ -105,6 +105,7 @@ def _run_proxy_analysis(env_names: List[str], score_function_names: List[str],
             all_predicates - {IsWet, IsDry},
             all_predicates - {IsClean, IsDirty},
             all_predicates - {OnTable},
+            all_predicates - {HoldingTop, HoldingSide},
```

Comment:
we could also try ablating each individually to ensure hill climbing would work with depth=0